ddtrace/contrib/django/utils.py

```diff
@@ -303,6 +305,23 @@ def _after_request_tags(pin, span, request, response):
             raw_uri = url
             if raw_uri and request.META.get("QUERY_STRING"):
                 raw_uri += "?" + request.META["QUERY_STRING"]
+
+            content_type = request.content_type
+            rest_framework = hasattr(request, "data")
+
+            req_body = None
+            if request.method in _BODY_METHODS:
```

Comment:
Is there a way for us to only do this if appsec is enabled?

What is the performance impact of this? Since we'd potentially be json decoding every single request, this could have a very large performance overhead. (stdlib json is quite slow).

The overhead might be worth it, but probably only if appsec is enabled.